### ISSUE TITLE
The result image shows higher accuracy and lower loss on Validation than Training (CSJ)

### ISSUE THREAD

**Issue by takeofuture** [espnet/espnet#3233]
I am training CSJ (Pure CSJ Corpus)
It takes very long time with 8GB GPU
But so far 25 epoches completed
I just see the images directory and check some image.
The total loss of Training is less than Validation which is normal.
But accuracy of validation is higher than that of training,
CTC loss and ATT loss of Validation  is smaller than those of Training.
How do I interpret this ? 

![loss](https://user-images.githubusercontent.com/33204332/119174223-2b683680-ba1d-11eb-9058-351e38caed65.png)
![loss_att](https://user-images.githubusercontent.com/33204332/119174225-2c00cd00-ba1d-11eb-9346-b971f6d2e494.png)
![loss_ctc](https://user-images.githubusercontent.com/33204332/119174227-2c996380-ba1d-11eb-81bb-392de5ed3675.png)
![acc](https://user-images.githubusercontent.com/33204332/119174231-2c996380-ba1d-11eb-828b-c5787da279a3.png)

**Comment by sw005320** [comment]
> But accuracy of validation is higher than that of training,
> CTC loss and ATT loss of Validation is smaller than those of Training.

This often happens when we use specaugment and dropout, which make training more difficult.

> The total loss of Training is less than Validation which is normal.

Actually, this sounds very strange to me.
Are you using espnet1?

**Comment by mthrok** [comment]
How are the metrics computed for training / validation?

When a dataset is large, and the model improves a lot through one epoch, if the accuracy and loss for training is averaged through epoch, while the accuracy and loss for evaluation is computed at the end of epoch, the evaluation metrics are computed with much better model, so this happens.

**Comment by takeofuture** [comment]
I am using ESPNET2 (No Kaldi Installed). But it seems both Training and Validation getting better from spech 1 to 25
I will wait next few weeks and evaluate the decoding.
CSJ uses the aligned training data(Start Time and Duration of the word sound included in the label). 
I am not sure ESPNET2 also uses the information as well.

Thank you!

**Comment by kan-bayashi** [comment]
It seems fine since we apply only specaug for the training data.
